### ISSUE TITLE
Ambiguity over dateTimeTimeZone.dateTime time zone

### ISSUE THREAD

**Issue by philipstratford** [microsoftgraph/microsoft-graph-docs-contrib#477]
What's not clear to me from this documentation is whether the value in `dateTimeTimeZone.dateTime` is UTC, with the value in `dateTimeTimeZone.timeZone` to be used to adjust it to be displayed for the time zone with which it was originally created, or whether `dateTimeTimeZone.dateTime` represents the time for the time zone specified in `dateTimeTimeZone.timeZone`.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e773d204-1933-45d8-17b2-5868787e25ca
* Version Independent ID: 9d5477ea-fe33-6234-94cf-fdfe3c3ecb9e
* Content: [dateTimeTimeZone resource type - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/resources/datetimetimezone?view=graph-rest-1.0)
* Content Source: [api-reference/v1.0/resources/datetimetimezone.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/v1.0/resources/datetimetimezone.md)
* Product: **outlook**
* Technology: **microsoft-graph**
* GitHub Login: @Harini84
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by BillyCottrell** [comment]
I agree with @philipstratford, it really is unclear on what dateTime actually is, and I even understand less as to why the timezone has to be separate from the dateTime as most dates have a zone at the end of the date instead, as it is being used with createdDateTime and other similar fields: "2023-06-21T13:59:41.8288286Z".
After all, if the timezone can be something different then UTC and dateTime itself is in the specified timezone and not UTC then having to convert this is basically impossible in JavaScript. Since, it makes it really hard to use the Date object and requires people to use the "moment-timezone" library to initialize a date with that timezone.
If it is not possible to make a start date with timezone offset included at the end maybe consider a separate property within the start and end properties called fullDate that has the timezone offset included.
